src/api/index.js

```diff
@@ -198,6 +219,19 @@ export async function setPhoneNumberForUser(uid, phoneNumber) {
   await firestore.collection('users').doc(uid).update({ phoneNumber });
 }
 
+export async function setProtestEditsForUser(user, protestId) {
+  await firestore
+    .collection('users')
+    .doc(user.uid)
+    .update({
+      edits: [...new Set([...(user.edits || []), protestId])],
```

Comment:
If you want to push the new protestsId into the edits list, I would use the following - https://firebase.google.com/docs/reference/js/firebase.firestore.FieldValue#arrayunion

It has been done similarly here - https://github.com/guytepper/1km.co.il/blob/5cf936c7e2d8ad021b1e20e04ec0c078366069ca/src/api/index.js#L350

Reply:
> Each specified element that doesn't already exist in the array will be added to the end.

According to the docs we don't need to use a `Set` to keep the array unique in firestore.